docs/QTIP.md

```diff
@@ -0,0 +1,25 @@
+# QTIP
+
+## What?
+
+QTIP is an MsQuic specific concept that allows MsQuic to exercise the QUIC protocol over XDP using TCP packets.
+
+## Why?
+
+Certain hardware / networks / cloud environments are optimized for TCP traffic. Instead of pushing for those environments to also optimize for UDP, QTIP allows us to leverage existing hardware with MsQuic for cheap.
+
+## How?
+
+
+To use QTIP, you first must be running XDP.
```

Comment:
It could be useful to say what happens if you set the option, but XDP is not available. Does setting the option fail? Is the setting ignored?

Reply:
Sounds good, will add in the next iteration.